acceptance/setup/aio/pre-suite/020_AIO_Workarounds.rb

```diff
@@ -1,9 +1,10 @@
 step "(PUP-4004) Set permissions on puppetserver directories that currently live in the agent cache dir"
+vardir = master.puppet('master')['vardir']
 %w[reports server_data yaml bucket].each do |dir|
-  on master, "install --directory /opt/puppetlabs/puppet/cache/#{dir}"
+  on master, "install --directory #{vardir}/#{dir}"
 end
-on master, "chown -R puppet:puppet /opt/puppetlabs/puppet/cache"
-on master, "chmod -R 750 /opt/puppetlabs/puppet/cache"
+on master, "chown -R puppet:puppet #{vardir}"
+on master, "chmod -R 750 #{vardir}"
```

Comment:
@er0ck could you remove this part, this workaround was removed in  https://github.com/puppetlabs/puppet/commit/482250f2c3032d483565755566ad1e695521d4f1